serveradmin/serverdb/models.py

```diff
@@ -426,6 +442,9 @@ def clean(self, *args, **kwargs):
                 raise ValidationError(
                     _('intern_ip must not be null'), code='missing value')
 
+            if type(self.intern_ip) not in [IPv4Interface, IPv6Interface]:
```

Comment:
Why not having this check inside `inet_to_python` already? It would makes sense to check it in the function too already.

Reply:
Because this is only the case for the special `intern_ip`. From what I could see `inet` attributes were always already transformed and I did not want to add this extra check for inet attribues. Once we remove `intern_ip` this would not be necessary anymore and this would be gone with the removal of this code block.